src/frontend/shortlist/src/views/CategorizeView.vue

```diff
@@ -110,6 +119,7 @@ export default {
             this.removeTopCard();
             console.log("DELETE school");
             // TODO set current_trashed in db
+            this.sendASchoolCardToTrash(this.myRecommendations[0].id, true);
```

Comment:
I think this needs to go _before_ line 119 which calls removeTopCard, otherwise won't it trash the next card, not the one that was removed.

Reply:
> * One question about order in CategorizeView.
> * Could you update the branch with develop? (below the checks)
> 
> And just to be extra clear, this doesn't actually delete anything from the database, it only marks the recommendation as "seen". Then since getting recommendations filters for only unseen schools it will not show up again in the stack.

Thanks for this clarification.